hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1547,13 +1547,21 @@ public static NetworkGetExecutionTimeQuery toPbj(
 
     public static DeterministicThrottle.UsageSnapshot fromPbj(ThrottleUsageSnapshot snapshot) {
         final var lastDecisionTime = snapshot.lastDecisionTime();
-        return new DeterministicThrottle.UsageSnapshot(
-                snapshot.used(), Instant.ofEpochSecond(lastDecisionTime.seconds(), lastDecisionTime.nanos()));
+        if (lastDecisionTime == null) {
+            return new DeterministicThrottle.UsageSnapshot(snapshot.used(), null);
+        } else {
+            return new DeterministicThrottle.UsageSnapshot(
+                    snapshot.used(), Instant.ofEpochSecond(lastDecisionTime.seconds(), lastDecisionTime.nanos()));
```

Comment:
We have `HapiUtils.asInstant()` to convert a `Timestamp` to an `Instant`.

Reply:
Updated.